dashboard/app/views/home/_levelbuilder.html.haml

```diff
@@ -10,3 +10,4 @@
     %li= link_to 'Helper Libraries', libraries_path
     %li= link_to 'Gallery', 'projects/public'
     %li= link_to 'Foorm Surveys (preview)', 'foorm/forms/editor'
+    %li= link_to '(BETA) Sprite Management', 'sprites/sprite_upload'
```

Comment:
our sprite management only works for spritelab right now, so we should call that out in the UI somewhere. that could either be here ("(BETA) Spritelab Sprite Management") or on the `sprite_upload` page itself

Reply:
Me, trying to decide how to spell ~~spritelab~~, ~~SpriteLab~~, ~~Spritelab~~, ~~spriteLab~~, Sprite Lab: https://media.giphy.com/media/l0CLThEZp4OtNNsli/giphy.gif

Reply:
hahahaha it's the ultimate question. i'm trying to standardize on Sprite Lab for marketing and external usage, and Spritelab/spritelab for code (because of an interesting point jessie made -- if we standardized on spriteLab in code, we'd have spriteLab/SpriteLab in js and sprite_lab in ruby)